src/nbl/asset/utils/CHLSLCompiler.cpp

```diff
@@ -353,6 +355,8 @@ std::string CHLSLCompiler::preprocessShader(std::string&& code, IShader::E_SHADE
 
 core::smart_refctd_ptr<ICPUShader> CHLSLCompiler::compileToSPIRV(const char* code, const IShaderCompiler::SCompilerOptions& options) const
 {
+    m_defaultIncludeFinder->addSearchPath("nbl/builtin/hlsl/jit", core::make_smart_refctd_ptr<CJITIncludeLoader>());
```

Comment:
read this again, https://github.com/Devsh-Graphics-Programming/Nabla/pull/605#discussion_r1392652518

There should be NO changes to `CHLSLCompiler`, only to `ILogicalDevice` and its default compiler set and include finder